elasticdl/python/master/task_dispatcher.py

```diff
@@ -11,6 +11,8 @@
 from elasticdl.python.common.constants import TaskExecCounterKey
 from elasticdl.python.common.log_utils import default_logger as logger
 
+MAX_TASK_RETRIES = 3
```

Comment:
Start variable name with underscore if it's only used internally

Reply:
Yeah! Done